### ISSUE TITLE
option for groupCollapsed?

### ISSUE THREAD

**Issue by anorudes** [fcomb/redux-logger#1]
It would be nice
https://developer.mozilla.org/en-US/docs/Web/API/Console/groupCollapsed

**Comment by istarkov** [comment]
+1 i've set this in real project and i also need `console.groupCollapsed`

**Comment by theaqua** [comment]
:+1: will release today.

**Comment by theaqua** [comment]
Hey @anorudes @istarkov, it's little bit difficult than I expected.
`Redux.applyMiddleware(...middleware)` looks for function, not it's call. So I wanted pass `logger(collapsed)` but then I realized I can't do this.

I have 2 solutions:
1. `applyMiddleware(logger.bind(null, true))` for pass `true` as argument
2. export 2 functions: `logger.default` and `logger.collapsed`.

Both solutions looks weird as for me. Maybe you have another ideas?

**Comment by istarkov** [comment]
mb like this

```javascript
function logger(options) {
  if ('getState' in options) {
     return (next) => (action) => blabla
  } else {
     // parse options
    // ...
    return ({getState}) => blabla
 }
}
```

**Comment by rt2zz** [comment]
+1 for having logger take options. In fact I am sure there will only be more options in the future, so you may as make it a createLogger function like devtools and others do

**Comment by ForbesLindesay** [comment]
:+1: for adding support for options.  I think @istarkov's approach looks good, and mirrors the way that most browserify transforms work.

I really need a way to have this log something more sensible when running server side in universal/isomorphic apps

**Comment by theaqua** [comment]
It will be released in 1.0.0 soon (https://github.com/fcomb/redux-logger/pull/5#issuecomment-131544593).

**Comment by theaqua** [comment]
Released!
https://github.com/fcomb/redux-logger/releases/tag/1.0.0
